packages/autocomplete/src/autocomplete.vue

```diff
@@ -218,6 +227,7 @@
         this.activated = false;
       },
       handleKeyEnter(e) {
+        if (this.blurOnSelect) { this.$refs.input.ignoreNextBlur(false); }
```

Comment:
_Info_ since the mouse cursor never click when using the keyboard to make the selection, the input is never unfocused. Thus, we do not need to ignore the next blur event.